src/BoundsInference.cpp

```diff
@@ -199,6 +201,32 @@ bool is_fused_with_others(const vector<vector<Function>> &fused_groups,
     return false;
 }
 
+class Inliner : public IRMutator {
```

Comment:
nit: worth inserting a comment here about why this is necessary in addition to the previously existing inliner elsewhere